PolyDeploy/Components/WebAPI/ActionFilters/InWhitelist.cs

```diff
@@ -57,6 +58,16 @@ public override void OnActionExecuting(HttpActionContext actionContext)
                 EventLogManager.Log("AUTH_EXCEPTION", EventLogSeverity.Info, null, ex);
             }
 
+            if (!authenticated)
```

Comment:
Perhaps we could have a comment explaining that we're bypassing IP whitelist for correctly configured users. The code is clear, but it doesn't fit in with the rest of the commenting style.

Reply:
@can-martin-kennish I added `// If IP is not whitelisted, check if API user can bypass whitelist`, let me know if there's any further changes you'd like to see